src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -445,6 +445,31 @@ const std::pair<Matrix, Matrix>& boundary_interpolation_matrices(
     size_t num_points) noexcept;
 // @}
 
+// @{
+/*!
+ * \brief Interpolates values from the boundary into the volume, which is needed
+ * when applying time derivative or Bjorhus-type boundary conditions in a
+ * discontinuous Galerkin scheme.
+ *
+ * Assumes that the logical coordinates are \f$[-1, 1]\f$. The first element of
+ * the pair is interpolation from \f$\xi=-1\f$ and the second from \f$\xi=1\f$.
+ * Specifically,
+ *
+ * \f{align*}{
+ *   &\ell^{\mathrm{Gauss-Lobatto}}_{0}(\xi_j^{\mathrm{Gauss}}) \\
+ *   &\ell^{\mathrm{Gauss-Lobatto}}_{N}(\xi_j^{\mathrm{Gauss}})
+ * \f}
```

Comment:
I did not find this math helpful... there's no `=`, and I don't know what is being described... what's going on?